onnxruntime/core/session/inference_session.h

```diff
@@ -132,8 +132,12 @@ class InferenceSession {
   explicit InferenceSession(const SessionOptions& session_options,
                             const Environment& session_env);
 
-#if !defined(ORT_MINIMAL_BUILD)
+  explicit InferenceSession(const SessionOptions& session_options,
+                            const Environment& session_env,
+                            onnxruntime::concurrency::ThreadPool* intra_op_thread_pool,
```

Comment:
1. Can these parameters be named as external_?
2. Also, we need documentation to justify the addition of this overload. Please state explicitly that it was added for WinML only.

Reply:
Sure